src/pages/settings/Security/TwoFactorAuth/Steps/VerifyStep.js

```diff
@@ -0,0 +1,136 @@
+import React, {useEffect} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {ScrollView, View} from 'react-native';
+import * as Session from '../../../../../libs/actions/Session';
+import styles from '../../../../../styles/styles';
+import Button from '../../../../../components/Button';
+import Text from '../../../../../components/Text';
+import ONYXKEYS from '../../../../../ONYXKEYS';
+import TextLink from '../../../../../components/TextLink';
+import Clipboard from '../../../../../libs/Clipboard';
+import FixedFooter from '../../../../../components/FixedFooter';
+import * as Expensicons from '../../../../../components/Icon/Expensicons';
+import PressableWithDelayToggle from '../../../../../components/Pressable/PressableWithDelayToggle';
+import TwoFactorAuthForm from '../TwoFactorAuthForm';
+import QRCode from '../../../../../components/QRCode';
+import expensifyLogo from '../../../../../../assets/images/expensify-logo-round-transparent.png';
+import CONST from '../../../../../CONST';
+import StepWrapper from '../StepWrapper/StepWrapper';
+import useTwoFactorAuthContext from '../TwoFactorAuthContext/useTwoFactorAuth';
+import useLocalize from '../../../../../hooks/useLocalize';
+import {defaultAccount, TwoFactorAuthPropTypes} from '../TwoFactorAuthPropTypes';
+
+const TROUBLESHOOTING_LINK = 'https://community.expensify.com/discussion/7736/faq-troubleshooting-two-factor-authentication-issues/p1?new=1';
+
+function VerifyStep({account = defaultAccount}) {
+    const {translate} = useLocalize();
+
+    const formRef = React.useRef(null);
+
+    const {setStep} = useTwoFactorAuthContext();
+
+    useEffect(() => {
+        Session.clearAccountMessages();
+    }, []);
```

Comment:
Coming from https://github.com/Expensify/App/issues/28937. Not really a bug but more like an improvement. We should have cleared the errors on the clean up function as well.